csharp/Platform.Numbers/Bit.cs

```diff
@@ -112,31 +157,79 @@ public static int GetLowestPosition(ulong value)
         /// <para>.</para>
         /// <para>.</para>
         /// </summary>
+        /// <typeparam name="T">
+        /// <para>Number type.</para>
+        /// <para>Тип числа.</para>
+        /// </typeparam>
+        /// <param name="x">
+        /// <para>The number on which the shift operation will be performed.</para>
+        /// <para>Число над которым будет производиться операция смещения.<para>
+        /// </param>
+        /// <param name="y">
+        /// <para>Number of bits to shift.</para>
+        /// <para>Количество бит на которые выполнить смещение.<para>
+        /// </param>
         /// <returns>
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>The changed value of x.</para>
+        /// <para>Изменённое значение x.</para>
         /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static T ShiftRight<T>(T x, int y) => Bit<T>.ShiftRight(x, y);
 
         /// <summary>
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>Performs a partial shift of a specified number of bits from source to target.</para>
```

Comment:
Not `shift` but `write`. `source to target` can be replaced to `source number to target number`.